__tests__/mint-tokens.test.js

```diff
@@ -1,13 +1,14 @@
+import { SendTransaction } from '@hathor/wallet-lib';
 import TestUtils from './test-utils';
 
 const walletId = 'stub_mint_tokens';
 
 describe('mint-tokens api', () => {
-  beforeAll(async () => {
+  beforeEach(async () => {
     await TestUtils.startWallet({ walletId, preCalculatedAddresses: TestUtils.addresses });
   });
 
-  afterAll(async () => {
+  afterEach(async () => {
     await TestUtils.stopWallet({ walletId });
```

Comment:
These have changed because the only mint authority was being marked as "selected_as_input" and another is not being created (due to how the http mock works) so we have to restart the wallet on every test